common/src/waiting.rs

```diff
@@ -0,0 +1,28 @@
+use std::sync::mpsc::channel;
+
+use codec::Decode;
+use log::{error, info};
+use substrate_api_client::ApiResult;
+
+use crate::Connection;
+
+pub fn wait_for_event<E: Decode + Clone, P: Fn(E) -> bool>(
+    connection: &Connection,
+    event: (&str, &str),
+    predicate: P,
+) -> anyhow::Result<E> {
+    let (module, variant) = event;
+    info!("[+] Creating event subscription {}/{}", module, variant);
```

Comment:
What about we give up on the convention of adding `[+]` to each log line? It originates from `substrate-api-client` code and made more sense when this code was in the `e2e-client` crate. Besides now we can run locally e2e tests with `./scripts/run_e2e.sh` which sets `RUST_LOG=aleph-e2e-client=info`, that effectively strips out this `[+]` log lines from `substrate-api-client`, so one don't need to grep on `[+]` anymore.

Reply:
Good idea, I'll try also adding a `target:` in this module.

Reply:
Done.